tests/api/entities/languages.test.js

```diff
@@ -0,0 +1,50 @@
+import 'should'
+import { publicReq } from '#tests/api/utils/utils'
+
+const endpoint = '/api/entities?action=languages'
+
+// The following tests require to have language entities indexed locally.
+// This can be done by running `npm run indexation:wikidata:load-languages-from-query`
+describe('entities:languages:search', () => {
+  it('should find a language by its label in the requested language', async () => {
+    const res = await publicReq('get', `${endpoint}&search=Korsische&lang=de`)
+    res.languages.should.be.an.Array()
+    const language = res.languages[0]
+    language.uri.should.equal('wd:Q33111')
+  })
+
+  it('should find a language by its label in another language', async () => {
+    const res = await publicReq('get', `${endpoint}&search=Korsische&lang=fr`)
+    res.languages.should.be.an.Array()
+    const language = res.languages[0]
+    language.uri.should.equal('wd:Q33111')
+  })
+
+  it('should find a language by its language code', async () => {
+    const res = await publicReq('get', `${endpoint}&search=hat&lang=zh`)
```

Comment:
is the `lang` param necessary ? if not, removing it make the test clearer on what is necessary to pass the test

```suggestion
    const res = await publicReq('get', `${endpoint}&search=hat`)
```

Reply:
removed in c51fc9f63